tests/models/time_series_transformer/test_modeling_time_series_transformer.py

```diff
@@ -52,10 +52,10 @@ def __init__(
         prediction_length=7,
         context_length=14,
         cardinality=19,
-        embedding_dimension=5,
+        embedding_dimension=4,
```

Comment:
Any reason this is changed to 4? With the new projection layer, it would work without changing this right?

Reply:
yes I believe so... let me revert it